### DISCUSSION TITLE
Issue template config file only works with .yml extension, not .yaml

### DISCUSSION THREAD

**Discussion by theprobugmaker** [community/community#36812] [Projects and Issues]
I noticed that if I create a `config.yaml` file inside the `ISSUE_TEMPLATE` it doesn't work, it only works with the `.yml` extension.

I'm creating a project and following a few standards and recommendations and one of them is from the official YAML website: https://yaml.org/faq.html which suggests that the official extension is `.yaml`.

Is it possible to enable compatibility with both `.yaml` and `.yml`? Thank you!

I'm not sure if that is the right category.

**Comment by ossanna16** [comment]
Thank you for asking a good question @theprobugmaker, I'm glad you found our community 🙂

**Comment by calmondev** [comment]


**Reply by ossanna16** [reply]
@calmonr I don't quite understand your question. Can you please explain further?

**Reply by calmondev** [reply]
> @calmonr I don't quite understand your question. Can you please explain further?

I'm sorry, wrong question.

**Reply by ossanna16** [reply]
That's ok 🙂

**Comment by vincerubinetti** [comment]
Was about to post a new question, glad someone else noticed this!

`.yaml` should really be supported in any and all places that `.yml` is supported. They are supposed to be interchangeable, and perhaps more importantly, [`.yaml` is the way recommended by YAML themselves](https://yaml.org/faq.html):

> Is there an official extension for YAML files?
> Please use ".yaml" when possible.

@theprobugmaker May I suggest editing the title to "_Issue template `config` file only works with `.yml` extension, not `.yaml`_" for better visibility and clarity? Maybe someone at GitHub will fix it.

**Reply by theprobugmaker** [reply]
Thank you for the suggestion. I will change the title. I hope they change that.

cc @ossanna16

**Comment by matifali** [comment]
Ran into this today. https://github.com/coder/coder/pull/16354

**Reply by michael-hawker** [reply]
Yup, me too, this is still an issue. 😢

**Comment by Ailrun** [comment]
This is still an issue.
